### PR TITLE
Token payment

### PR DESCRIPTION
**Pull request by ashleybarrett**

### CODE REVIEW

File: src/JudoDotNetXamarin/JudoDotNetXamarin.csproj.orig [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (JamesGoodwin) [inline src/JudoDotNetXamarin/JudoDotNetXamarin.csproj.orig]: Don't think we need this file checked in

File: src/JudoDotNetXamarin/Pages/CardEntryPage.xaml_BACKUP_49115.cs [hunk 2]
```diff
@@ -0,0 +1,269 @@
+using System;
```
Comment (JamesGoodwin) [inline src/JudoDotNetXamarin/Pages/CardEntryPage.xaml_BACKUP_49115.cs]: Left over file from merge?

File: src/JudoDotNetXamarin/JudoDotNetXamarin.csproj.orig (hunk 1 above)
Comment (ashleybarrett) [inline src/JudoDotNetXamarin/JudoDotNetXamarin.csproj.orig]: removed

File: src/JudoDotNetXamarin/Pages/CardEntryPage.xaml_BACKUP_49115.cs (hunk 2 above)
Comment (ashleybarrett) [inline src/JudoDotNetXamarin/Pages/CardEntryPage.xaml_BACKUP_49115.cs]: removed

**Pull request merged**